Source/diablo.cpp

```diff
@@ -284,6 +297,9 @@ bool LeftMouseCmd(bool bShift)
 
 bool LeftMouseDown(int wParam)
 {
+	lastLeftMouseButtonAction = MOUSEACTION_OTHER; //Fluffy
+	lastLeftMouseButtonTime = SDL_GetPerformanceCounter();

```

Comment:
Use `SDL_GetTicks` instead of `SDL_GetPerformanceCounter` as it is not available on all targets, and not needed for what is happening here.